doc/INSTALL.md

```diff
@@ -77,16 +86,22 @@ _If you need to reconfigure the URLs for GitHub.com, GitHub:Enterprise, or Rally
+## SSL configuration
+During the installation process, we create a root CA and a set of self-signed certs to suport SSL on your Takeout install. You will either need to (1) replace our certificates with your own certificates signed by an already trusted CA, or (2) trust our root CA in your system.
 
 _NOTE: If you do not take one of the following actions, some features in Waffle may not function properly._
 
-#### Trusting our root CA
-A root CA was generated during the install process and saved on the host machine at `/etc/waffle/ca-certificates/waffle-root-ca.crt`. You will need to ask a system administrator to trust this certificate in your system.
-
 #### Using your own certificates
-Self-signed certificates were created during the install process and saved on the host machine at `/etc/waffle/nginx/certs`. If you look in that directory, you will see 4 files: `<hostname>.crt`, `<hostname>.key`, `*.<hostname>.crt`, and `*.<hostname>.key`. You may provide your own certs by:
+This is the prefered way to handle SSL for Waffle Takeout. To use your own certificates, you simply overwrite our self-signed certificates with your own trusted certificates and restart the service. The self-signed certificates live on the host machine at `/etc/waffle/nginx/certs`. If you look in that directory, you will see 4 files: `<hostname>.crt`, `<hostname>.key`, `*.<hostname>.crt`, and `*.<hostname>.key`. You may provide your own certs by:
 
 1. Generate certificates signed by a trusted CA for both `<hostname>.crt` and `*.<hostname>.crt` where `hostname` is the hostname of the host machine running waffle. Those names must be exact.
-2. Overwrite the existing files in `/etc/waffle/nginx` with the certificates you created. You must upload both the `.crt` and the `.key` files.
+2. Overwrite the existing files in `/etc/waffle/nginx/certs` with the certificates you created. You must upload both the `.crt` and the `.key` files into that directory.
+3. Run `sudo service waffle restart` to restart Waffle Takeout using the new certs.
+
+#### Trusting our root CA
+A root CA was generated during the install process and saved on the host machine at `/etc/waffle/ca-certificates/waffle-root-ca.crt`. This is the certificate you need to trust. You have 2 options for trusing it:
+
+1. Often times, system administraitors can trust certificates within your domain environment. This approach will trust Waffle Takeout's self-signed certs for all machines bound to your domain. You need to ask a system administrator if this is an option for you and provide them with the certificate at `/etc/waffle/ca-certificates/waffle-root-ca.crt`.
+2. Each user of Waffle Takeout needs to trust the certificate on their own machine. Each will need to obtain the certificate from `/etc/waffle/ca-certificates/waffle-root-ca.crt` and then follow these instructions:
```

Comment:
Can we add something like "This is obviously not ideal to have every user trust certificates individually, we highly recommend installing your own trusted certificates if you cannot trust our self signed certificates at a domain level."